test/lib/bamboo/mailer_test.exs

```diff
@@ -447,6 +453,26 @@ defmodule Bamboo.MailerTest do
     end
   end
 
+  describe "interceptors" do
+    setup do
+      Application.put_env(:bamboo, __MODULE__.MailerWithInterceptors, @mailer_config)
+    end
+
+    test "must apply interceptor and send email if not intercepted" do
```

Comment:
Thank you for adding these tests for `deliver_now` 🎉  

I like that we're testing two conditions: when the interceptor modifies the email (this test) and when the interceptor blocks the email (the next one). 👍